zcash_transparent/src/keys.rs

```diff
@@ -67,7 +67,7 @@ impl From<TransparentKeyScope> for ChildNumber {
 /// A child index for a derived transparent address.
 ///
 /// Only NON-hardened derivation is supported.
-#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd)]
```

Comment:
- Why is this needed?
- Addition should be documented in the changelog.
- This should derive `Ord` as well if we're deriving `PartialOrd`.